### ISSUE TITLE
Segmentation fault (core dumped) - Ubuntu 16.04

### ISSUE THREAD

**Issue by VanitarNordic** [tzutalin/labelImg#53]
Hi,

I can not run the executable file because of this error:

`Segmentation fault (core dumped)`

**Comment by tzutalin** [comment]
Interesting. I only tried it on ubutnu14.04.  Are you running it on a 32-bit and 64-bit CPU?

**Comment by VanitarNordic** [comment]
64bit- Ubuntu 16.04. I run the software from the python code and it worked flawlessly but the executable no.

**Comment by tzutalin** [comment]
@VanitarNordic ,
   Are you running on i386 amd64?

**Comment by VanitarNordic** [comment]
x64 intel processor

**Comment by arnaudmiribel** [comment]
Same problem here. 

But I have to point I've had issues running the `make all`. Output was : 

```
*** No rule to make target 'all'. Stop.
```

**Comment by VanitarNordic** [comment]
@arnaudmiribel 

Make sure the path is correct when you run make all

**Comment by arnaudmiribel** [comment]
I do a `cd` in the `labelImg` main directory though.. 

<img width="653" alt="make all doesnt work" src="https://cloud.githubusercontent.com/assets/7164864/23856165/dfc46968-07f8-11e7-959b-2e3fed00786e.png">

**Comment by VanitarNordic** [comment]
Where is your make file? you should do in that path where related files exist

**Comment by arnaudmiribel** [comment]
Oh well I thought I could only use binaries but seems like by themselves it doesn't work.
I instead cloned the repo and was able to `make` of course. Now it works. Thanks for your help.

**Comment by hejiangda** [comment]
try sudo ./labelImg

**Comment by aharris02** [comment]
@hejiangda your comment solved it for me.

**Comment by tzutalin** [comment]
Duplicate #61

**Comment by skanwwal** [comment]
Hi, 

I have the same problem. Initially I thought it is some problem related to JDK on ubuntu 16 system as Matlab software keep giving me an error of graphics issue. I am trying to process some images using GAMMA software in terminal and it gives me this segmentation problem. I am a beginner. I'd really appreciate if some can provide any assistance. Thank u

**Comment by appdotpy** [comment]
This fixed it for me
(python 3.5.3, virtualenv, Debian 4.9.110, pip3, lxml, pyqt5)

Was trying to load labelImg within a virtualenv 'venv'. Got the segfault.

I opened an interactive python3 terminal and ran:

`from PyQt5 import QtCore`
Segmentation fault.

I opened a new terminal with "sudo python3" and repeated the command. This time it worked properly.

So I created a brand new virtualenv:

`virtualenv venv2 --system-site-packages`

I did another git clone && re-ran the installation instructions for Python 3 + Qt5. This time I did not get the segmentation fault and labelImg loaded properly.

In my case it seems like the segmentation fault is the result of a permissions problem when trying to import subpackages from PyQt5. That being said, 'venv2' referenced the same global packages (pyqt5 is a global package) as 'venv'. Perhaps there were conflicting packages in 'venv'.
